.rubocop_todo.yml

```diff
@@ -58,17 +47,27 @@ RSpec/ContextMethod:
 RSpec/ContextWording:
   Enabled: false
 
-# Offense count: 80
+# Offense count: 85
```

Comment:
This number has increased but I don't think it should have. Maybe a rebase?

Reply:
I think it might have, it's 84 even with a rebase